### ISSUE TITLE
matching stubs not including query strings

### ISSUE THREAD

**Issue by chochkov** [oesmith/puffing-billy#21]
Hey, I wonder what the intention is behind not matching query strings in stubs?

Example:

```ruby
proxy.stub('http://myapi.com/user/?country=FOO')
# is considered the same as:
proxy.stub('http://myapi.com/user/?anything=FOO')
# and generally the same as:
proxy.stub('http://myapi.com/user/')
```

I find this unexpected, but I see that this is the specified behaviour https://github.com/oesmith/puffing-billy/blob/master/spec/lib/billy/proxy_request_stub_spec.rb#L31

**Comment by gkopylov** [comment]
@chochkov I create this PR to fix this issue https://github.com/oesmith/puffing-billy/pull/93
So you can configure Billy like this to avoid stripped query params:
Billy.configure do |c|
c.strip_query_params = false
end

**Comment by ronwsmith** [comment]
Addressed in v0.5.0
